### PR TITLE
Fix notebooks in inference track

### LINKED ISSUE #973: Fix notebook `examples/custom_transform_and_model.ipynb` to fit current hierarchy of models
### 🚀 Feature Request

Now notebook `examples/custom_transform_and_model.ipynb` isn't in accordance with hierarchy of models (I thought it should fail tests).

### Proposal

Fix `LGBMMultiSegmentModel` to be like `CatBoostMultiSegmentModel`.

### Test cases

Notebook works.

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Mr-Geekman**
## Before submitting (must do checklist)

- [x] Did you read the [contribution guide](https://github.com/tinkoff-ai/etna/blob/master/CONTRIBUTING.md)?
- [x] Did you update the docs? We use Numpy format for all the methods and classes.
- [x] Did you write any new necessary tests?
- [x] Did you update the [CHANGELOG](https://github.com/tinkoff-ai/etna/blob/master/CHANGELOG.md)?

## Proposed Changes
Look #973.

## Closing issues
Closes #973.

### CODE REVIEW

**Review by martins0n — CHANGES_REQUESTED** [review]
comments in notebook

**Comment by Mr-Geekman** [comment]
<p>Even for per-segment type?</p>

---

 <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/974/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

File: .binder/requirements.txt [hunk 1]
```diff
@@ -1 +1 @@
-etna[all] @ git+https://github.com/tinkoff-ai/etna.git@master
+etna[all] @ git+https://github.com/tinkoff-ai/etna.git@issue-974
```
Comment (martins0n) [inline .binder/requirements.txt:1]: why do we need this?

Comment (Mr-Geekman) [reply .binder/requirements.txt:1]: I will remove it, it is for the case when I want to run it on clean machine in binder (some warnings aren't present where).

**Comment by Mr-Geekman** [comment]
<p>I will fix random seed inside model.</p>

---

 <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/974/discussion/'>View entire conversation</a> on ReviewNB<div id="RAICReplyForNotification-ReviewNBCommentContext-DoNotDelete"></div>

**Review by martins0n — APPROVED** [review]

**Pull request merged**